observability/WFLY-17144-add-micrometer-support.adoc

```diff
@@ -0,0 +1,87 @@
+ = Add Micrometer Support to WildFly
+:author:            Jason Lee
+:email:             jasondlee@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+WildFly already supports basic metrics in the base WildFly system, as well as MicroProfile Metrics in the WildFly MicroProfile configuration. In the Java space, however, the library of choice, and the emerging de facto standard for Java developers is https://micrometer.io[Micrometer].  This RFE will add support for Micrometer to WildFly, providing an alternative to the WildFly Metrics subsystem, as well as providing a replacement for MicroProfile Metrics, which is slated for removal from WildFly under another JIRA.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-17144
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/EAP7-1686
+* https://issues.redhat.com/browse/EAP7-1805
+* https://issues.redhat.com/browse/WFLY-17138
+* https://issues.redhat.com/browse/EAP7-1975
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+* [X] Engineering
+* [X] QE
```

Comment:
You need to pick only one!

Reply:
+1. This choice basically declares what help you expect from the QE contributors working with you.

Either choice, the engineer participates in developing and running tests.